src/lib/prisma-session-store.ts

```diff
@@ -220,7 +225,68 @@ export class PrismaSessionStore<M extends string = 'session'> extends Store {
       this.logger.warn(
         `Attempt to destroy non-existent session:${String(sid)} ${String(e)}`
       );
-      if (callback) defer(callback, e);
+      if (callback) return defer(callback, e);
```

Comment:
@wSedlacek - I've assumed, here, that the prior lack of `return` was a bug. Does that seem right?